gomi: add page (#16166)
tldr-pages/tldr: pages/common/gomi.md

# gomi

> Unlike `rm`, which permanently deletes files, `gomi` moves them to the trash, allowing you to easily restore files.
> See also: `rm`.
> More information: <https://github.com/babarot/gomi>.

> - Delete safely specific files or folders (no need for flag `-rf`):

Reviewer: ```suggestion
- Safely delete specific files or folders:
```

> `gomi {{path/to/file1 path/to/folder ...}}`

Reviewer: ```suggestion
`gomi {{path/to/file1 path/to/file2 path/to/folder1 path/to/folder2 ...}}`
```

> - Restore one or more files, going back:

Reviewer: What does this mean?

Author: ![2025-04-13-081846_hyprshot](https://github.com/user-attachments/assets/22a5f512-f61f-46dd-8804-08d7a56a09f4)

gomi --restore opens a menu that let's you select the items deleted that you wnat to restore

Reviewer: ```suggestion
- Open an interactive menu to restore one or more files:
```

`gomi {{[-b|--restore]}}`

> - Remove files older than a specific time ([d]ay, [w]eek, [m]onth, [y]ear):

Reviewer: ```suggestion
- Remove files from the trashcan older than a specific time ([d]ay, [w]eek, [m]onth, [y]ear):
```
This probably needs clarification. Otherwise someone might think it's talking about the current working directory.

> - Remove files older than a specific time ([d]ay, [w]eek, [m]onth, [y]ear):

Reviewer: ```suggestion
- Remove files that have been in the trashcan longer than the specified time ([d]ay, [w]eek, [m]onth, [y]ear):
```
This feels like it needs clarification

> - Remove files older than a specific time (day, week, month, year):

Reviewer: ```suggestion
- Remove files older than a specific time ([d]ay, [w]eek, [m]onth, [y]ear):
```
@sebastiaanspeck I think the mnemonics here were fitting

> `gomi --prune={{1d|1w|1m|1y|...}}`

Reviewer: ```suggestion
`gomi --prune {{1d|1w|1m|1y|...}}`
```
Is the `=` necessary?

Author: I didn't this it would work, I followed the github docs, but yeah works fine without the equal sign. Thanks

> - Remove orphaned `.trashinfo` files (which contain the info of the deleted files, when thay have been deleted and where, a metadata file, that is abandoned and not useful anymore):

Reviewer: ```suggestion
- Remove orphaned `.trashinfo` files:
```

> `gomi --prune={{orphans}}`

Reviewer: ```suggestion
`gomi --prune {{orphans}}`
```
